src/util/virutil.c

```diff
@@ -277,6 +277,67 @@ virParseVersionString(const char *str, unsigned long *version,
     return 0;
 }
 
+int
+virCHVersionString(const char *str, unsigned long *version,
```

Comment:
Looks like the right place for this function would be `src/ch/ch_conf.c` or some other CH related area. Patching this file seems not appropriate as it's unrelated to CH.

Thanks

Reply:
Moved it to ch_conf.c as discussed. Thnx :)